molecule/config_upload/converge.yml

```diff
@@ -27,6 +27,7 @@
         cartridge_app_config_path: './configs/config.yml'
         cartridge_app_config_upload_mode: null
         cartridge_app_config_upload_url: 'http://vm1:3301/admin/config'
+        cartridge_app_config_upload_http_timeout: 30
```

Comment:
Why do you explicitly set default value?

Reply:
Due the bug in Ansible: cartridge_app_config_upload_http_timeout value will be share for neighboring calls